src/crimson/osd/osd.cc

```diff
@@ -341,24 +341,28 @@ seastar::future<> OSD::_add_me_to_crush()
   if (!local_conf().get_val<bool>("osd_crush_update_on_start")) {
     return seastar::now();
   }
-  const double weight = [this] {
+  auto get_weight = [this] () {
```

Comment:
nit, /()// as if a lambda does not accepts any parameters the parameter list can be dispensed.